### PR TITLE
Two trivial changes (typo + info message)

### PR DESCRIPTION
**Pull request by PabloCastellano**

### CODE REVIEW

File: pmb/flasher/frontend.py [hunk 1]
```diff
@@ -62,6 +63,13 @@ def kernel(args):
     else:
         logging.info("(native) flash kernel " + flavor)
         pmb.flasher.run(args, "flash_kernel", flavor)
+    logging.info("You will get an IP automatically assigned to your "
+                 "USB interface shortly.")
+    logging.info("Connect to the telnet session and type your LUKS "
+                 "password to boot postmarketOS:")
```
Comment (MartijnBraam) [inline pmb/flasher/frontend.py]: This should probably only be printed if luks is actually enabled.

Comment (PabloCastellano) [inline pmb/flasher/frontend.py]: That was my first approach but args.full_disk_encryption is only available when `pmbootstrap install` is run. This is `pmbootstrap flasher`

Comment (ollieparanoid) [inline pmb/flasher/frontend.py]: you could add `(not necessary when full disk encryption disabled):` at the end of the sentence.

**Comment by PabloCastellano** [comment]
Changes done

**Pull request merged**